tests/WebAppCallsMicrosoftGraph/Startup.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Identity.Web;
+using Microsoft.Identity.Web.TokenCacheProviders.InMemory;
+using Microsoft.Identity.Web.UI;
+
+namespace WebAppCallsMicrosoftGraph
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddSignIn(Configuration)
```

Comment:
@jmprieur , @jennyf19 Question about AddSignIn. Would the developer need to do this even when they configure EasyAuth and have signed in the user? 
If they need to configure sign in again with client creds, what is the benefit of EasyAuth? Does the integration still make sense?

Reply:
I have a question too, regarding AddSignin and the other service collection extenions, there is a motive to we are not using PKCE? After all, its the actual recomentation from the last [BCP](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-14).

Reply:
IT's not yet implemented in MSAL.NET, @peterson4ts. When it is, we'll use it.
If you don't need to call a Web API, feel free to use it.

Reply:
> IT's not yet implemented in MSAL.NET, @peterson4ts. When it is, we'll use it.
> If you don't need to call a Web API, feel free to use it.

@jmprieur  Okay. I'm already using my implementation of AddSignIn with PKCE. But there is something that I still don't understand, for example, when using the AddWebAppCallsProtectedWebApi method it injects the ResponseType with the "code id_token" that would not allow me to use the PKCE with the challenge and still use the MSAL?